scripts/ci/tags.yaml

```diff
@@ -46,52 +46,50 @@
 
 bluetooth:
   files:
-    - zephyr/drivers/bluetooth/
-    - zephyr/subsys/bluetooth/
-    - zephyr/include/zephyr/bluetooth/
-    - zephyr/samples/bluetooth/hci_ipc/
-    - nrfxlib/softdevice_controller/include/
-    - nrfxlib/softdevice_controller/lib/
-    - nrfxlib/mpsl/include/
-    - nrfxlib/mpsl/lib/
-    - nrf/subsys/bluetooth/
-    - nrf/include/bluetooth/
-    - nrf/subsys/nrf_rpc/
-    - nrf/subsys/mpsl/
-    - nrf/drivers/mpsl/
-    - nrf/samples/bluetooth/
-    - nrf/tests/bluetooth/
-    - nrf/tests/subsys/bluetooth/
-    - nrf/samples/nrf5340/multiprotocol_rpmsg/
     - modules/crypto/mbedtls/
     - modules/crypto/oberon-psa-crypto/
     - modules/crypto/tinycrypt/
+    - nrf/cmake/
+    - nrf/drivers/mpsl/
+    - nrf/include/bluetooth/
+    - nrf/samples/bluetooth/
+    - nrf/samples/nrf5340/multiprotocol_rpmsg/
+    - nrf/subsys/bluetooth/
+    - nrf/subsys/mpsl/
+    - nrf/subsys/nrf_rpc/
     - nrf/subsys/nrf_security/
+    - nrf/sysbuild/
+    - nrf/tests/bluetooth/
+    - nrf/tests/subsys/bluetooth/
     - nrfxlib/crypto/
+    - nrfxlib/mpsl/include/
+    - nrfxlib/mpsl/lib/
+    - nrfxlib/softdevice_controller/include/
+    - nrfxlib/softdevice_controller/lib/
+    - zephyr/cmake/
+    - zephyr/drivers/bluetooth/
+    - zephyr/include/zephyr/bluetooth/
+    - zephyr/samples/bluetooth/hci_ipc/
```

Comment:
You may also add dependency onto `nrf/applications/ipc_radio` here I think (IIRC it's used by default for Bluetooth on multi-core platforms). The `zephyr/samples/bluetooth/hci_ipc/` is no longer used by default so that maybe we could remove HCI IPC sample dependency too. @dchat-nordic, could you confirm?

Reply:
The ipc_radio is used by default in almost all NCS. I think it should be added to bluetooth and ieee802154 (zigbee, thread etc) tags.

The hci_ipc is still used by a couple samples AFAIK: 2 matter samples and SUIT samples. I don't really know how they are tagged in this context. In my opinion if that's possible to leave those samples checked for hci_ipc it's fine to remove the hci_ipc for everything else.

Reply:
Added     - nrf/applications/ipc_radio/ for mentioned tags.